stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/data/user/UserRepository.kt

```diff
@@ -52,6 +52,10 @@ class UserRepository(context: Context) {
         prefs.edit().clear().apply()
     }
 
+    fun clearUserCommit() {
```

Comment:
Can we add a `commit` or `async` flag to the existing `clearUser` function with a predefined default value to keep previous behavior?

Reply:
Actually we don't need this funtion